src/amo/server/base.js

```diff
@@ -261,6 +261,12 @@ function baseServer(
         store.dispatch(setRequestId(thisRequestId));
       }
 
+      let isAnonymousRequestOrPage;
```

Comment:
I wasn't sure what this was initially, also a default value wouldn't hurt I guess. Could you think of a better name?

Reply:
Looking at this more closely... I could simplify this by not depending on `isAnonymousPage`, but just the presence of the token. That would mean authenticated users looking at a page where `isAnonymousPage` is `true` wouldn't generate a cached response, but I think that's perfectly ok.

Reply:
https://github.com/mozilla/addons-frontend/pull/10190/commits/f74ef02a9b55e9552bf39ae55ce0f421685956c7 does this